lib/nrf5/common/temp_common.c

```diff
@@ -0,0 +1,41 @@
+/** @addtogroup temp_file
+
+@author @htmlonly &copy; @endhtmlonly 2018 
+Gerrit Maus <funk@maus.xyz>
+Luong Le <novercy@live.com>
+
+This library supports the TEMP in the NRF52 series.
```

Comment:
this is the space where you have room to write what TEMP actually is.  otherwise you might as well have just left this out.